test/gtest/coll_score/test_score_str.cc

```diff
@@ -71,7 +71,6 @@ UCC_TEST_F(test_score_str, check_range_1)
     EXPECT_NE(UCC_OK,
               check_range(score, UCC_COLL_TYPE_ALLTOALL, UCC_MEMORY_TYPE_HOST,
                           RLIST({RANGE(64, 256, 10)})));
-    ucc_coll_score_free(score);
```

Comment:
Why free is removed here?

Reply:
because score variable is already freed in line 69

Reply:
i see. now check_range at line 72 accesses score after free. lets remove score_free at 69 and keep it at 74.